### PR TITLE
Add kik messenger

### PR DESCRIPTION
**Pull request by danieltaylor**

### CODE REVIEW

File: _data/communication.yml [hunk 1]
```diff
@@ -8,3 +8,9 @@ websites:
       goog: Yes
       authy: No
       doc: http://windows.microsoft.com/en-au/windows/two-step-verification-faq
+
+    websites:
```
Comment (computmaxer) [inline _data/communication.yml]: There should only be one `websites:` declaration per section file, and instead the lines below this should be un-indented one and line up with the `-name Skype` above. I'll be able to merge once you fix this. Thanks!

**Comment by computmaxer** [comment]
Also there seems to be a problem with the icon. It appears to be a text file with the .png extension. It should instead be an actual image/binary file.

**Comment by danieltaylor** [comment]
@computmaxer Oops, sorry about that, fixed now.  As for the icon, I was just messing around with it in my own fork (I guess I accidentally submitted it and communication.yml before they were ready), but I couldn't actually figure out how to upload the icon to the repo.  How would I go about doing this?

**Comment by jdavis** [comment]
Also, there's an error with the build: https://travis-ci.org/jdavis/twofactorauth/builds/21070460

**Comment by jdavis** [comment]
Closing for clean up, feel free to resubmit this if you fix the issues.

**Pull request closed without merging**